### ISSUE TITLE
Watchtower issue

### ISSUE THREAD

**Issue by Xitro01** [jgeusebroek/docker-spotweb#53]
I keep getting watchtower errors while trying to pull your repo:

```
Could not do a head request for "jgeusebroek/spotweb:latest", falling back to regular pull.
Reason: registry responded to head request with "404 Not Found", auth: "not present"
```

I read somewhere that it has something to do with buildx that they are adding provenance information in manifests per default now.
https://github.com/docker/buildx/releases/tag/v0.10.0

Hopefully you can look into this, thanks in advance.

**Comment by jgeusebroek** [comment]
Thanks, I'm facing this issue myself. I will have a look when I have time.

**Comment by jgeusebroek** [comment]
Fixed with 747c6bdedb9d3339f1b1944e0d14ace5073c1823.
